setup.py

```diff
@@ -23,15 +23,15 @@
 Python-level packaging using distutils.
 """
 
-from distutils.core import setup
 import ipalib
+from ipasetup import ipasetup  # pylint: disable=relative-import
 
 
-setup(
+ipasetup(
     name='freeipa',
+    doc=__doc__,
     version=ipalib.__version__,
-    license='GPLv3+',
-    url='http://freeipa.org/',
+    scripts=['ipa'],
```

Comment:
The `ipa` script is in fact a client script, so I guess it shouldn't be packaged with `ipaserver`.